src/haddock/modules/defaults.yaml

```diff
@@ -94,3 +94,14 @@ self_contained:
     main installation.
   group: 'execution'
   explevel: guru
+clean:
```

Comment:
If the cleaning is not a module that the  user needs to add at the end, it might be safer to make the default to false for the time being. This parameter should be specified in all our examples cfg file, possibly as commented out, but to make clear it can be used.